### PR TITLE
Fixes wrong parameter type in function `recreateIntlObjects` that causes generate_from_arb to exit

### PR DESCRIPTION
**Pull request by lukas-h**
In some of my .arb files, I had the issue, that the `@` at the beginning of a key resulted in an error:
```
Unhandled exception:
type '_InternalLinkedHashMap<String, dynamic>' is not a subtype of type 'String'
```

Here is an example of the .arb:
```
...
"super_nice_key": "Super Nice Key",
"@super_nice_key": {
        "type": "text"
},
...
```

Why the problem exists:
The values in the json file, which here are `"Super Nice Key"`, or `{ "type": "text" }`, are not always decoded as type `String`, but when passed to the function `recreateIntlObjects(String id, String data)` as `data`, they are expected to be always of type `String`.

What I did is, I changed the type of parameter  `data` to `dynamic`, so it supports the decoded Map from the arb file too.

BTW inside the function, there was already an explicit type check for `LiteralString`, so `dynamic` is not removing any type safety here.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
```
Comment (lukas-h) [inline .github/ISSUE_TEMPLATE/bug_report.md:1]: @LorenzMaehringer please revert the changes here. We don’t want these issue templates as parts of our PR

**Comment by mosuem** [comment]
This has been fixed as part of the null safety migration! But thank you.
